lib/Swagger/src/Discussion/Api/SitesApi.php

```diff
@@ -193,16 +181,13 @@ public function createSiteWithHttpInfo($site, $x_wikia_schwartz = null)
                 $queryParams,
                 $httpBody,
                 $headerParams,
-                '\Swagger\Client\Discussion\Models\HalResponse'
+                '\Swagger\Client\Discussion\Models\SiteHalResponse',
+                '/sites'
             );
 
-            return array($this->apiClient->getSerializer()->deserialize($response, '\Swagger\Client\Discussion\Models\HalResponse', $httpHeader), $statusCode, $httpHeader);
+            return array($this->apiClient->getSerializer()->deserialize($response, '\Swagger\Client\Discussion\Models\SiteHalResponse', $httpHeader), $statusCode, $httpHeader);
         } catch (ApiException $e) {
             switch ($e->getCode()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)